test/test_drakrun.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+@pytest.fixture(scope="session")
+def pytest_installed(drakmon_vm):
+    """ Ensure that pytest is installed """
+    drakmon_vm.run(". /opt/venvs/drakrun/bin/activate && pip install pytest==6.2.2 pytest-steps==1.7.3")
+
+@pytest.fixture(scope="session")
+def drakrun_test_dir(pytest_installed, drakmon_vm):
+    """ Find location of tests """
+    res = drakmon_vm.run(
+    """
+    . /opt/venvs/drakrun/bin/activate &&  \
+    python -c "import os; import drakrun.test.conftest; print(os.path.dirname(drakrun.test.conftest.__file__))"
+    """)
+
+    return res.stdout.strip()
+
+def test_lvm(drakmon_vm, drakrun_test_dir, pytest_installed):
```

Comment:
`pytest_installed` is being invoked when we are using `drakrun_test_dir` fixture, ig these are redundant here

Reply:
Yes, but we have to know the path